app/src/main/java/ch/epfl/sweng/erpa/services/GCP/LazyGameStream.java

```diff
@@ -0,0 +1,45 @@
+package ch.epfl.sweng.erpa.services.GCP;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.List;
+import java.util.Map;
+
+import ch.epfl.sweng.erpa.model.Game;
+import retrofit2.Call;
+
+public class LazyGameStream extends LazyAsyncStream<Game> {
```

Comment:
How do you update? (e.g. when the user pulls the view back, he should see the new games created which are now on top)

Reply:
Say server collects (according to the requested sort criteria) games with ID [ 4, 6, 8, 10, 12...]
The client asks  for a page of length 3: [4, 6, 8]
a new game (2) now qualifies for the filter
The clients requests the next page of length 3, the server responds [8, 10, 12]
We have a duplicate 8, how do you handle this?